pkg/csi-common/utils.go

```diff
@@ -18,15 +18,66 @@ package csicommon
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 	"strings"
+	"sync"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	"github.com/kubernetes-csi/csi-lib-utils/protosanitizer"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	typedv1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/homedir"
 	"k8s.io/klog/v2"
 )
 
+const (
+	NameSpaceEnvVar   = "KUBERNETES_NAMESPACE"
+	PodNameEnvVar     = "POD_NAME"
+	CSIEventSourceStr = "blob-csi-driver"
+)
+
+const (
+	// Driver "Normal" event Reason list
+	NodeStagingVolume      = "NodeStagingVolume"
+	NodeStagedVolume       = "NodeStagedVolume"
+	NodeUnStagingVolume    = "NodeUnStagingVolume"
+	NodeUnStagedVolume     = "NodeUnStagedVolume"
+	NodePublishingVolume   = "NodePublishingVolume"
+	NodePublishedVolume    = "NodePublishedVolume"
+	NodeUnPublishingVolume = "NodeUnPublishingVolume"
+	NodeUnPublishedVolume  = "NodeUnPublishedVolume"
+	CreatingBloContainer   = "CreatingBloContainer"
```

Comment:
can we add the missing `b` in `Blo`? or was there a reason it was omitted?

Reply:
Ooops! typo